aws/data_source_aws_ecs_cluster_test.go

```diff
@@ -59,3 +81,40 @@ data "aws_ecs_cluster" "default" {
   cluster_name = "${aws_ecs_cluster.default.name}"
 }
 `, acctest.RandInt())
+
+var testAccCheckAwsEcsClusterDataSourceConfigContainerInsights = fmt.Sprintf(`
+resource "aws_ecs_cluster" "default" {
+  name = "default-%d"
+  setting {
+    name = "containerInsights"
+    value = "enabled"
+  }
+}
+
+resource "aws_ecs_task_definition" "mongo" {
```

Comment:
Nit: Since the `aws_ecs_task_definition` and `aws_ecs_service` resources are not necessary for testing the `aws_ecs_cluster` data source, we can omit them.